system-administrators/deployment/network-deployment.md

```diff
@@ -17,6 +17,15 @@ First step is to get the MedCo Deployment latest release at each node. Adapt `${
 ```bash
 export MEDCO_SETUP_DIR=~/medco MEDCO_SETUP_VER=v2.0.1
 git clone --depth 1 --branch ${MEDCO_SETUP_VER} https://github.com/ldsec/medco.git ${MEDCO_SETUP_DIR}
+
```

Comment:
This part here is docker specific and not really medco specific. So it makes sense to have it, but it should be a part of the "common problems" page.